internal/crypto/helpers.go

```diff
@@ -179,3 +185,66 @@ func GenerateRSAPublicKey(tb testing.TB) []byte {
 	publicKeyPEM := pem.EncodeToMemory(publicKeyBlock)
 	return publicKeyPEM
 }
+
+func GetPaddedString(str string) []byte {
+	return []byte(RightPad2Len(str, "=", 32))
+}
+
+// encrypt string to base64 crypto using AES
+func Encrypt(key []byte, text string) string {
+	plaintext := []byte(text)
```

Comment:
slight improvements:

- don't shadow with `text` variable name (template/text), rename
- is there a maximum length we can trim the incoming text to? linter on L205 correctly points out an overflow is possible due to lack of length checks; if we trim it to `in[:255]` we can avoid those

Reply:
I agree with the shadowing of the "text" variable => updated to string

Regarding the second comment this is ancient code that I didn't want to touch/revise now as it's out of the scope of the current ticket which already brings out a lot of changes.